db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Answer.delete_all
+ApplicationRecord.delete_all
+Keyword.delete_all
+Officer.delete_all
+Question.delete_all
+Rso.delete_all
+
+rso.create!(name: 'Association for Computer Machinery',
```

Comment:
Will do. I was reading and in the RoR reference manual you gave me they say to use bang creations because they're better. It didn't give much more explanation than that.